lib/common/socket/evloop.c.h

```diff
@@ -358,7 +405,12 @@ struct st_h2o_evloop_socket_t *create_socket(h2o_evloop_t *loop, int fd, struct
 {
     struct st_h2o_evloop_socket_t *sock;
 
+#ifdef _WIN32
+    u_long nonblock = 1;
+    ioctlsocket(fd, FIONBIO, &nonblock);
+#else
     fcntl(fd, F_SETFL, O_NONBLOCK);
+#endif
```

Comment:
For setting the nonblocking flag, please create a wrapper function for setting the nonblock flag in `h2o/include/socket.h` (marked as static the same way as other functions to be inlined are), and call the function from other places.